topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,657 @@
+  - emmaleith
+  - mariecrane
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Metaproteomics involves characterization of community level expression of microbial proteins from an environmental 
+or clinical sample. Metaproteomics data is primarily used to determine the functional status of the microbiome under 
+study along with its taxonomic composition. The Galaxy-P team published a software suite named [metaQuantome](https://www.mcponline.org/content/18/8_suppl_1/S82) to enable quantitative and statistical analysis and visualization of functional, taxonomic expression as well as functional and 
+taxonomy interaction. metaQuantome leverages peptide level quantitative information to analyze the taxonomic, functional 
+expression within the microbial community in different conditions.
+
+<p align="center">
+  <img width="850" height="500" src="../../images/microbiome.png" alt="Microbiome" title="Microbiome" />
+</p>
+
+ 
+metaQuantome offers differential abundance analysis, principal components analysis, and clustered heat map visualizations, 
+across multiple experimental conditions. metaQuantome, an open source tool, is available via command line and also 
+accessible via Galaxy platform for reproducible analysis. As a first step for metaQuantome analysis, metaproteomics 
+data needs to be made compatible for subsequent analysis. With this in mind, we have developed a metaQuantome data 
+generation workflow tutorial that ill help users generate inputs for metaQuantome analysis.
+
+<p align="center">
+  <img width="850" height="500" src="../../images/metaquantomeworkflow.png" alt="Workflow" title="Workflow">
+</p>
+
+To demonstrate the use of the data creation workflow, we have used a thermophilic biogas reactor dataset wherein municipal 
+food waste and manure is digested to generate methane gas. After one round in the reactor, the microbial community was 
+simplified and enriched via serial dilution. This inoculum was then transferred to a solution of cellulose from Norwegian 
+Spruce and incubated at 65°C. Triplicate samples were taken in a time series from 0 to 43 hours after inoculation and mass 
+spectrometry data was acquired on a Q-Exactive (Thermo) mass spectrometer. For this training, we have chosen two time points-8 hour and 33 hour.
+
+<p align="center">
+  <img width="800" height="300" src="../../images/biogasdataset.png" alt="Dataset" title="Dataset">
+</p>
+
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# **Pretreatments**
+
+The first step in a tutorial is to get the data from the zenodo link provided and making sure that it is in the correct format.
+
+## *Get data*
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files ( 6 MZML files, a Protein FASTA file and Experimental Design file) from [Zenodo](https://zenodo.org/badge/DOI/10.5281/zenodo.4037137.svg)](https://doi.org/10.5281/zenodo.4037137) or from
```

Comment:
You can leave this line the way it was generated by planemo. Since you filled in the Zenodo URL in the metadata, it will be automatically used here. This makes it easier if you update the data later, only have to change the metadata. Same for the `topic_name` and `page.title`, anything inside the double curly braces tags like `{{ x }}, is a jekyll variable and you don't have to change.

So change step 2 back to:
```
> 2. Import the files ( 6 MZML files, a Protein FASTA file and Experimental Design file) from [Zenodo]({{ page.zenodo_link }}) 
>    or from the shared data library (`GTN - Material` -> `{{ page.topic_name }}` -> `{{ page.title }}`)
```